Tests/DistributedClusterTests/Plugins/ClusterSingleton/ClusterSingletonPluginTests.swift

```diff
@@ -89,4 +131,20 @@ final class ClusterSingletonPluginTests: SingleClusterSystemXCTestCase {
             print("Hello!")
         }
     }
+    
+    distributed actor ActorWithId {
+        let customId: String
+        
+        init(
+            actorSystem: ActorSystem,
+            id: String
+        ) {
+            self.actorSystem = actorSystem
+            self.customId = id
+        }
+        
+        distributed func getId() -> String { 
```

Comment:
```suggestion
        distributed func getID() -> String { 
```

big `ID` everywhere please :)